cachito/web/migrations/versions/cb6bdbe533cc_add_env_var_types.py

```diff
@@ -0,0 +1,52 @@
+"""Add environment variables types
+
+Revision ID: cb6bdbe533cc
+Revises: 615c19a1cee1
+Create Date: 2020-06-22 14:56:53.584293
+
+"""
+from alembic import op
+from sqlalchemy.engine.reflection import Inspector
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "cb6bdbe533cc"
+down_revision = "b46cf36806d7"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    with op.batch_alter_table("environment_variable", schema=None) as batch_op:
+        # Drop previous unique_constraint. Since the constraint was not originally added with a
```

Comment:
Can you just look at the automatic name used by Postgres and fix the old migration to set the name explicitly instead?

Reply:
IMO that's too brittle. We should instead setup explicit [naming conventions](https://alembic.sqlalchemy.org/en/latest/naming.html), and create another migration script to "rename" them all.

Reply:
We have successfully done this in the past:
* https://github.com/release-engineering/cachito/commit/3bfee393276abc0d5b0c59f18b0340d3129d2689
* https://github.com/release-engineering/cachito/commit/dbb205747d089cfadd4418741a5aa09a1942d778